app/Http/Controllers/Legacy/Web/SignupsController.php

```diff
@@ -55,7 +55,7 @@ public function show($id)
                 'signup_id' => $signup->id,
                 // @TODO - We could probably grab campaign and user info from API as well.
                 'campaign' => $campaign,
-                'user' => $user->toArray(),
+                // 'user' => $user->toArray(),
```

Comment:
this will be commented back in, just needed to comment this out on my local bc I have some weird settings!